metron-interface/metron-config/package.json

```diff
@@ -7,10 +7,11 @@
   },
   "angular-cli": {},
   "scripts": {
-    "build": "./node_modules/angular-cli/bin/ng build -prod",
+    "build": "./node_modules/@angular/cli/bin/ng build --prod",
```

Comment:
It can also be written as `ng build --prod`

Reply:
Thanks for pointing this out! I think the original path may have been leftover from the original project generation with the old version of angular cli. I have since removed it.